schemas/fingerprint-server-api.yaml

```diff
@@ -1499,6 +1509,13 @@ components:
           type: boolean
           description: Flag indicating whether the request came from a device with location spoofing enabled.
           example: false
+    SuspectScoreResult:
+      type: object
+      properties:
+        result:
+          type: integer
+          description: Integer value that determines fraudulent actor probability based on certain Smart Signals patterns.
```

Comment:
@petrpalata feel free to suggest better description

Reply:
[0;inf) - this is not normalized score

Reply:
I got his as a highlight in the documentation

> Suspect Score is an easy way to integrate Smart Signals into your fraud protection work flow. It is a weighted representation of all Smart Signals present in the payload that helps identify suspicious activity.

Feel free to use it if you think it's a better description. We also have a [dedicated docs page](https://dev.fingerprint.com/docs/suspect-score) that shows the weights so it wouldn't hurt to include that in the description.

Reply:
Thanks, updated

Reply:
The description is better, but I still wouldn't know how to interpret the result, just by reading the description. Consider either: 

* Adding a `minimum` and `maximum` values, if that at least theoretically makes sense (but I guess they are platform dependant and could chang often?)
* Adding a link to https://dev.fingerprint.com/docs/suspect-score

Reply:
fixed